build/k8s/ingestor.yaml

```diff
@@ -149,7 +149,7 @@ spec:
           readinessProbe:
             httpGet:
               path: /readyz
-              port: 9090
```

Comment:
This doesn't seem right.  9091 shouldn't be reachable externally.

I'd suggest just adding `scheme: HTTPS` instead.

Reply:
I see, ok updated the PR.